flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/MiniClusterResource.java

```diff
@@ -49,15 +54,17 @@
  * Starts a Flink mini cluster as a resource and registers the respective
  * ExecutionEnvironment and StreamExecutionEnvironment.
  */
-public class MiniClusterResource extends ExternalResource {
+public class MiniClusterResource extends ExternalResource implements JobExecutor {
```

Comment:
Why do we need to implement `JobExecutor`?

Reply:
so that we can pass the MiniClusterResource to `TestEnvironment.setAsContext()`